src/App.tsx

```diff
@@ -1,16 +1,33 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
+  const addNewMovie = ({
+    title, description, imgUrl, imdbUrl, imdbId,
+  }: Movie) => {
```

Comment:
It's better to destructure the value inside the function
```suggestion
  const addNewMovie = (movie: Movie) => {
    const { title, description, imgUrl, imdbUrl, imdbId } = movie;
```